nemo/collections/asr/parts/submodules/ctc_beam_decoding.py

```diff
@@ -439,6 +554,21 @@ class PyCTCDecodeConfig:
     hotword_weight: float = 10.0
 
 
+@dataclass
+class FlashlightConfig:
+    #kenlm_path: Optional[str] = None
+    lexicon_path: Optional[str] = None
+    nbest: int = 1
+    #beam_size: int = 16
+    beam_size_token: int = 16
+    beam_threshold: float = 20.0
+    #lm_weight: float = 0.2
+    #word_score: float = 0.2
```

Comment:
Are these two args not supported? If so, could you remove them?

Reply:
Removed